_inc/client/plans/plan-body.jsx

```diff
@@ -64,51 +68,187 @@ const PlanBody = React.createClass( {
+					{
+						includes( [ 'jetpack_premium', 'jetpack_premium_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Backups & Security Scanning' ) }</h3>
+								<p>{ __( 'Daily backup of all your site data with unlimited space, one-click restores, automated security scanning, and priority support (powered by VaultPress).' ) }</p>
+								{
+									this.props.isFetchingPluginsData ? '' :
+									this.props.isPluginInstalled( 'vaultpress/vaultpress.php' )
+									&& this.props.isPluginActive( 'vaultpress/vaultpress.php' ) ? (
+										<Button href="https://dashboard.vaultpress.com/" className="is-primary">
+											{ __( 'View your security dashboard' ) }
 										</Button>
-									</p>
-								</div>
-								: ''
-						}
-					</div>
-				);
-				break;
+									)
+									: (
+										<Button href={ 'https://wordpress.com/plugins/setup/' + this.props.siteRawUrl + '?only=vaultpress' } className="is-primary">
+											{ __( 'Configure VaultPress' ) }
+										</Button>
+									)
+								}
+							</div>
+						: ''
+					}
+
+					{
+						includes( [ 'jetpack_business', 'jetpack_business_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Backups & Security Scanning' ) }</h3>
+								<p>{ __( 'Real-time backup of all your site data with unlimited space, one-click restores, automated security scanning, one-click threat resolution, and priority support (powered by VaultPress).' ) }</p>
+								{
+									this.props.isFetchingPluginsData ? '' :
+									this.props.isPluginInstalled( 'vaultpress/vaultpress.php' )
+									&& this.props.isPluginActive( 'vaultpress/vaultpress.php' ) ? (
+										<Button href="https://dashboard.vaultpress.com/" className="is-primary">
+											{ __( 'View your security dashboard' ) }
+										</Button>
+									)
+									: (
+										<Button href={ 'https://wordpress.com/plugins/setup/' + this.props.siteRawUrl + '?only=vaultpress' } className="is-primary">
+											{ __( 'Configure VaultPress' ) }
+										</Button>
+									)
+								}
+							</div>
+						: ''
+					}
+
+					{
+						includes( [ 'jetpack_premium', 'jetpack_premium_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Video Hosting' ) }</h3>
+								<p>{ __( '13Gb of fast, optimised, and ad-free video hosting for your site (powered by VideoPress).' ) }</p>
+								{
+									this.props.isFetchingPluginsData ? '' :
+									this.props.isPluginInstalled( 'vaultpress/vaultpress.php' )
+									&& this.props.isPluginActive( 'vaultpress/vaultpress.php' ) ? (
+										<Button href="https://vaultpress.com/" className="is-primary">
+											{ __( 'TO DO: Text and link if unconfigured' ) }
```

Comment:
There is a TODO here, and please don't make strings like those translatable in case they get accidentially uploaded to translate.wordpress.org.